build/catalog/packages.js

```diff
@@ -38,7 +38,7 @@ module.exports = function (imports) {
         }))
         .pipe(stream.concat(function (guides) {
 
-          package.version = details._release;
+          package.version = /^[0-9]+\.[0-9]+/.test(details._release) ? details._release : '';
```

Comment:
the carbon elements package doesn't have a release version yet. Without this check, the version would be the SHA hash.